### PR TITLE
Doc: add third column with examples \o/

### PR DESCRIPTION
**Pull request by stifoon**
This is soooo cool.

### CODE REVIEW

File: documentation/slate/source/includes/apis.md [hunk 1]
```diff
@@ -52,12 +43,16 @@ ressource id:
 }
 ```
 
-The coverage id is *"regions": ["fr-idf"]* so you can ask Navitia on
-accessible local mobility services:
+> in this example, the coverage id is "regions": ["fr-idf"] 
+so you can ask Navitia on accessible local mobility services:
```
Comment (antoine-de) [inline documentation/slate/source/includes/apis.md:45]: I know it's not in the scope of your PR, but I think it's better to say `available` and not  `accessible`

File: documentation/slate/source/includes/apis.md [hunk 2]
```diff
@@ -83,8 +78,17 @@ accessible local mobility services:
         {"href": "https://api.navitia.io/v1/coverage/fr-idf/"}
     ]
 }
+
 ```
 
+Also known as `/coord` service.
```
Comment (antoine-de) [inline documentation/slate/source/includes/apis.md]: Hum I'm not sure I understand why this is there

File: documentation/slate/source/includes/apis.md [hunk 3]
```diff
@@ -266,10 +273,13 @@ Examples :
 
 ### Few exploration examples
 
-Response example for this request
-<https://api.navitia.io/v1/coverage/fr-idf/physical_modes>
+``` shell
+#request
+$ curl 'https://api.navitia.io/v1/coverage/sandbox/physical_modes' -H 'Authorization: 3b036afe-0110-4202-b9ed-99718476c2e0'
```
Comment (antoine-de) [inline documentation/slate/source/includes/apis.md]: token

File: documentation/slate/source/includes/apis.md [hunk 4]
```diff
@@ -308,6 +318,27 @@ chapter
 <a name="pt-objects"></a>Autocomplete on Public Transport objects 
 -----------------------------------------------------------------
 
+``` shell
+#request
+$ curl 'https://api.navitia.io/v1/coverage/sandbox/pt_objects?q=metro%204&type\[\]=line&type\[\]=route' -H 'Authorization: 3b036afe-0110-4202-b9ed-99718476c2e0'
```
Comment (antoine-de) [inline documentation/slate/source/includes/apis.md]: token

File: documentation/slate/source/includes/apis.md [hunk 5]
```diff
@@ -22,24 +22,15 @@ The only arguments are the ones of [paging](#paging).
...
 --------------------------------------
+``` shell
+#request
+$ curl 'https://api.navitia.io/v1/coord/2.376915;48.84675' -H 'Authorization: my-token'
 
-Also known as `/coord` service.
-
-Very simple service: you give Navitia some coordinates, it answers you
-where you are:
+#the request above will give you the right Navitia coverage, and a useful label
```
Comment (pbougue) [inline documentation/slate/source/includes/apis.md]: I would move this line above (replacing `request`)

File: documentation/slate/source/includes/apis.md (hunk 2 above)
Comment (stifoon) [inline documentation/slate/source/includes/apis.md]: From official documentation https://github.com/tripit/slate/wiki/Markdown-Syntax
"Code samples will appear in the dark area to the right of the main text. We recommend positioning code samples right under headers in your markdown file."
This sentence is so the first of the chapter. Above, it is the shell example

File: documentation/slate/source/includes/apis.md (hunk 5 above)
Comment (stifoon) [inline documentation/slate/source/includes/apis.md]: Commited

**Comment by stifoon** [comment]
Hey, poke @pbougue @antoine-de ;)

**Comment by stifoon** [comment]
gnagnagna http://stifoon.github.io/navitia

**Comment by stifoon** [comment]
And http://stifoon.github.io/navitia is updated FTW \o/
And for @antoine-de

File: documentation/slate/source/includes/welcome.md [hunk 6]
```diff
@@ -15,11 +15,20 @@ It provides the following services
...
+
+<aside class="notice">
+A fake token is used in examples below: be aware that this token is really limited. You should use yours to get better services on real data.
+</aside>
 
 Getting started
 ---------------
 
+``` shell
+$ curl 'https://api.navitia.io/v1/coverage/sandbox/' -H 'Authorization: 3b036afe-0110-4202-b9ed-99718476c2e0'
```
Comment (antoine-de) [inline documentation/slate/source/includes/welcome.md]: It is a bit abrupt as the first example. 
maybe we can add some explanation above ?

**Comment by antoine-de** [comment]
apart from my remark, it's ok for me

**Pull request merged**